src/pages/Board/component/CreateItemForm.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import {Button, Form, DatePicker, Input, InputNumber, Select} from 'antd'
+import {useTranslation} from 'react-i18next'
+import Item from '../../../models/Item'
+import moment from 'moment'
+import {FormComponentProps} from 'antd/es/form'
+import {Category} from '../../../models/Category'
+
+const {TextArea} = Input
+const {Option} = Select
+
+interface Props extends FormComponentProps {
+	isCateogryLoading: boolean
```

Comment:
probably we need a form prop here?

Reply:
The form prop is already in FormComponentProps